bciers/apps/reporting/src/app/components/facility/FacilityEmissionAllocationForm.tsx

```diff
@@ -125,7 +127,23 @@ export default function FacilityEmissionAllocationForm({
   navigationInformation,
   isPulpAndPaper,
   overlappingIndustrialProcessEmissions,
+  facilityType,
 }: Props) {
+  // If facility type is small or medium, add not applicable as an option
+  const modifiedEmissionAllocationSchema = emissionAllocationSchema;
+  if (
+    ["Small Aggregate", "Medium Facility"].includes(facilityType) &&
+    modifiedEmissionAllocationSchema.properties
+  ) {
+    const object = {
+      type: "string",
+      title: "Methodology",
+      enum: ["Not Applicable", "OBPS Allocation Calculator", "Other"],
+    };
+    modifiedEmissionAllocationSchema.properties.allocation_methodology =
+      object as RJSFSchema;
+  }
+
```

Comment:
For emission allocation, the "Not Applicable" methodology is available to any type of facility, according to the AC of this card. We can add the value straight into the `emissionAllocationSchema` instead. 

Then `facilityType` is not needed and it simplifies things in `FacilityEmissionAllocationPage`